src/lib/style/bootstrap4_backports/_overrides.scss

```diff
@@ -14,17 +14,20 @@
     }
   }
 
-  // change styling of dropdowns
-  %dropdown-active-state {
-    background: SRC.$primary-action-color;
-    color: white;
```

Comment:
This made dropdowns hard to configure. Getting rid of this didn't seem to cause a problem.